canonical-web-and-design/ubuntu.com
# Fix other 12 to 48TB storage bits

## Done

- updated from 12 to 48TB for storage in a few more places per https://trello.com/c/tEw3rGNM/82-due-soon-due-soon-change-ceph-allowance

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/features and http://0.0.0.0:8001/openstack/features, http://0.0.0.0:8001/legal/ubuntu-advantage-service-description and http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/ja and compare to the copy docs [uasd](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit#) and [openstack/features](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit#)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

Fixes #5381
